code/modules/power/singularity/particle_accelerator/particle_control.dm

```diff
@@ -150,9 +150,9 @@
 	..()
 	if(stat & NOPOWER)
 		active = FALSE
-		use_power = NO_POWER_USE
+		change_power_mode(NO_POWER_USE)
```

Comment:
This won't do anything since `NO_POWER_USE` is `0` and the proc immediately returns if `!use_type`. There are several other instances of this too.

Reply:
that needs to be a `isnull(use_type)` check, oops. Thank you for catching that.

Reply:
fixed